### PR TITLE
Added GH60 Satan ISO layout with split right shift and split backspace

### PR DESCRIPTION
**Pull request by dende**

### CODE REVIEW

File: keyboards/satan/satan.h [hunk 1]
```diff
@@ -138,6 +138,36 @@
...
+   * | 20    | 21| 22| 23| 24| 25| 26| 27| 28| 29| 2a| 2b| 2c| 2d|
+   * |-----------------------------------------------------------|
+   * | 30 | 31| 32| 33| 34| 35| 36| 37| 38| 39| 3a| 3b|  3d  | 3c|
+   * |-----------------------------------------------------------|
+   * | 40 | 41 | 42 |        45             | 4a | 4b | 4c | 4d  |
+   * `-----------------------------------------------------------'
+
+   swap 3c and 3d. 3c is right of 3d in reality
+ */
+#define KEYMAP_ISO_SPLITRSHIFT_SPLITBACKSPACE( \
```
Comment (drashna) [inline keyboards/satan/satan.h]: Could you use "LAYOUT" here instead, or add a define below to map it? 

We're depreciating KEYMAP in favor of LAYOUT.

Comment (dende) [reply keyboards/satan/satan.h]: Done!
<sub>The commits get squashed anyway, don't they? :sweat_smile:</sub>

Comment (drashna) [reply keyboards/satan/satan.h]: Awesome, and yup.  Usually. :)

**Comment by drashna** [comment]
Nice, thanks!

**Pull request merged**